lms/djangoapps/verify_student/models.py

```diff
@@ -1186,3 +1186,64 @@ def check_user_skipped_reverification_exists(cls, user, course_id):
             Boolean
         """
         return cls.objects.filter(user=user, course_id=course_id).exists()
+
+
+class CreditCourse(models.Model):
+    """Model for tracking the credit course."""
+
+    course_key = CourseKeyField(max_length=255, db_index=True, unique=True)
+
+
+class CreditProvider(models.Model):
+    """This model represents an institution that can grant credit for a course.
+    each provider is identified by unique id e.g ASU
+    """
+    provide_id = models.CharField(max_length=255, db_index=True, unique=True)
+    display_name = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
```

Comment:
I don't think you need `_at` appended to field names like `created` and `modified`.